python/ray/function_manager.py

```diff
@@ -396,8 +396,11 @@ def load_actor_class(self, job_id, function_descriptor):
             actor_methods = inspect.getmembers(
                 actor_class, predicate=is_function_or_method)
             for actor_method_name, actor_method in actor_methods:
-                method_descriptor = PythonFunctionDescriptor(
-                    module_name, actor_method_name, actor_class_name)
+                if actor_method_name == "__init__":
+                    method_descriptor = function_descriptor
```

Comment:
document why this if statement is needed

Reply:
OK